python-avd/pyavd/_eos_designs/structured_config/network_services/vlans.py

```diff
@@ -98,6 +98,18 @@ def _get_vlan_config(
             name=vlan.name,
             tenant=tenant.name,
         )
+        if vlan.address_locking.address_family.ipv4:
```

Comment:
we should not have address_family level in the input schema.

Reply:
removed!